.golangci.yml

```diff
@@ -31,3 +31,4 @@ linters:
     - lll
     - prealloc
     - funlen
+    - interfacer
```

Comment:
The linter is deprecated (https://github.com/mvdan/interfacer) - let's not keep it. Also is not the issue in this PR.